### ISSUE TITLE
Asset Maintenance Type

### ISSUE THREAD

**Issue by jmartins1** [grokability/snipe-it#10109]
### Snipe-IT Version

n/a

### Operating System

n/a

### Web Server

n/a

### PHP Version

n/a

### Is your feature request related to a problem? Please describe. A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]

Please reopen #7766 

### Describe the solution you'd like A clear and concise description of what you want to happen.

#7766

### Describe alternatives you've considered A clear and concise description of any alternative solutions or features you've considered.

 
jmartins1 commented on Jan 28, 2020
Please add the ability to modify the Asset Maintenance Types.

Example: If I need to replace an item, say a power cord, the asset maintenance type REPLACE is not available.

Thank you

@weekly-digest weekly-digest bot mentioned this issue on Feb 2, 2020
Weekly Digest (26 January, 2020 - 2 February, 2020) #7782
 Closed
@King-GO
 
King-GO commented on Feb 20, 2020
+1
for equipment, periodical "qualification" is required.

@stale
 
stale bot commented on Apr 22, 2020
Is this still relevant? We haven't heard from anyone in a bit. If so, please comment with any updates or additional detail.
This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. Don't take it personally, we just need to keep a handle on things. Thank you for your contributions!

@stale stale bot added the stale label on Apr 22, 2020
@stale
 
stale bot commented on Apr 30, 2020
This issue has been automatically closed because it has not had recent activity. If you believe this is still an issue, please confirm that this issue is still happening in the most recent version of Snipe-IT and reply to this thread to re-open it.

@stale stale bot closed this on Apr 30, 2020
@kreinartz83
 
kreinartz83 commented on May 21, 2020
+1, also it would be nice to have a few more features while doing asset maintenance
these are some of the things I could use

ability to add parts to maintenance, from consumables stock, or non stocked parts, (Part Number, Description, Cost, Quantity, etc ) as data fields
ability to track the person, or supplier performing the maintenance (Who, amount of time, cost per amount of time) as data fields
Ability to attach files to a maintenance for documentation such as invoices, pictures, calibration info, etc
Add a timestamp to the maintenance date so that it shows hours out of service instead of just days
ability to add custom fields to suit others needs
Basically i would like to be able to use the software as my repair shop software as well as my asset management software so that i can better track the full cost of my assets

@tehkillerbee
 
tehkillerbee commented 7 hours ago
You should consider reopening this issue, as we would like to add some more relevant maintenance types.

@NikoBach2
 
NikoBach2 commented 7 hours ago
+1 It would be nice with the possibility to add extra asset maintenance types.

**Comment by tarmoinno** [comment]
+1.

**Comment by sulekhahelpdesk** [comment]
+1

**Comment by Boba1982** [comment]
1

**Comment by jodylang** [comment]
+1

**Comment by snipe** [comment]
Duplicate of #7766
